src/types/Phone.ts

```diff
@@ -0,0 +1,16 @@
+export type Phone = {
```

Comment:
Please rewrite this type according to json:
https://github.com/mate-academy/product_catalog/blob/main/public/api/phones.json

Use **interface** instead **type**.